src/components/Onfido/BaseOnfidoWeb.tsx

```diff
@@ -59,11 +63,10 @@ function initializeOnfido({sdkToken, onSuccess, onError, onUserExit, preferredLo
             {
                 type: CONST.ONFIDO.TYPE.DOCUMENT,
                 options: {
-                    useLiveDocumentCapture: true,
```

Comment:
It was removed from lib some time ago https://documentation.onfido.com/sdk/web/#no--1100

Reply:
ok, but we should make sure isn't being used in Onfido SDK, because it can be an issue with TS typings but not the code itself.

Reply:
From what I understood [it was deprecated in 10.1.10](https://documentation.onfido.com/sdk/web/#no--1010---2022-12-08), but we are at `8.3.0`.

Reply:
@fabioh8010 `8.3.0.` is a version of `@onfido/react-native-sdk` which is used by mobile platforms.
The version of `onfido-sdk-ui` used by E/App for web is `13.1.0`, so it's deprecated.

Reply:
Thanks for explanation @VickyStash !